docs/tutorials/01-fee/07-test-app.md

```diff
@@ -105,15 +105,9 @@ hermes create channel --channel-version '{"fee_version":"ics29-1","app_version":
 ```
 
 This will create an incentivized IBC transfer channel between the two chains with the channel id `channel-0`, and channel version `{"fee_version":"ics29-1","app_version":"ics20-1"}`.
-Next recall that the Fee Middleware only pays fees on the source chain. That's why we should register `damian` and `charlie` as each other's counterparty on both chains.
-
-```bash title="Terminal 4"
-hermes fee register-counterparty-payee --chain mars --channel channel-0 --port transfer --counterparty-payee cosmos1vapwvcsr0m32ptal6z6g9hjctywrw4yzyf6y6v
-```
 
-```bash title="Terminal 4"
-hermes fee register-counterparty-payee --chain earth --channel channel-0 --port transfer --counterparty-payee cosmos1uu38gkyed0dte5f9xk20p8wcppulsjt90s7f8h
-```
+Next recall that the Fee Middleware only pays fees on the source chain. That's why we should register `damian` and `charlie` as each other's counterparty on both chains.
+Luckily, the relayer does this for us under the hood because we've enabled the `auto_register_counterparty_payee` option in the config file.
```

Comment:
This `auto_register_counterparty_payee` option is only mentioned here. Should it be mentioned somewhere else as well when setting up hermes config file?

Reply:
It is already added with [this](https://github.com/cosmos/ibc-go/pull/4979/files#diff-b84ca4e422c3888dcc4d8a5b55acdf7f96c340d26a522da1374d6f7447d0cce2L53). Docusaurus renders them as code snippets